tests/arg-changed-kind.spec.ts

```diff
@@ -1,45 +1,66 @@
 import { gql } from "./testkit";
 
-test("safe: adding a default value to existing non-nullable argument", () => {
+/** @note: This would risk breaking consumers if a new value is returned though... E.g. an int in this case */
```

Comment:
I'm not sure how we want to handle this in our breaking changes logic.
It's technically safe to make this change but could create issues if clients dont handle the new kind.
I think ideally, for a change like this, we'd request that clients approve the change once they handle the new type. Or maybe we consider this a dangerous change... Not sure atm.